src/Display/Modal/Modal.tsx

```diff
@@ -61,6 +68,25 @@ const Modal = (props: Props) => {
       document.removeEventListener('keydown', escapeKeyEventListener, false);
   }, [isVisible, onClose]);
 
+  React.useLayoutEffect(() => {
```

Comment:
Should we name this function, as there's already one useLayoutEffect hook called, would be better to know the responsibility for this.

Reply:
ok!